README.md

```diff
@@ -19,16 +19,32 @@ Once you have implemented the `fm` program (and tested it to the best of your ab
 
 ## Preprocessing
 
-*What preprocessing data do you store in files, and how?*
+We decided to make a new file with the same name as the previous file as a prefix, ending with '_prepro.txt' instead of '.fa' to connect the two. This preprocessing file contains a fasta-style header, followed by a line initiated with '#' containing the suffix array sa(x), a line with the dictionary for C after a '+' and finally a line initiated by '@' followed by a dictionary containing the table O.
```

Comment:
It is a good idea to name the preprocessed file something that depends on the input genome file (as otherwise you probably cannot link them). If you always put three lines in the preprocessed files, why do you need the special characters at the beginning of the lines, though?

(You might also want to have a look at [`pickle`](https://docs.python.org/3/library/pickle.html) if you plan to serialise things in Python in the future).